.github/workflows/regression_test.yml

```diff
@@ -62,5 +62,5 @@ jobs:
         pip install ${{ matrix.torch-spec }}
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
-        pip install .
+        python setup.py install
```

Comment:
do we need to update README? for building from source, is it `python setup.py developer` now? I spent some time to understand why `pip install -e .` does not work anymore and find this PR

Reply:
argh, lemme update now